src/main/resources/assets/susy/lang/en_us.lang

```diff
@@ -786,6 +786,8 @@ metaitem.standard_catalyst_bed_cracking.name=Cracking Catalyst Bed
 # Machine Translations
 susy.ocean_pumper.full=Tanks are Full!
 susy.ocean_pumper.drainrate=Current Pumping Rate: %s
+susy.wrongbiome=Machine can not work in this biome!
```

Comment:
are these langkeys necessary?

Reply:
not really, but I would like to avoid merge conflict when I'm done with Ocean Pump Rework